pkg/controller/migplan/gvk.go

```diff
@@ -0,0 +1,62 @@
+package migplan
+
+import (
+	migapi "github.com/fusor/mig-controller/pkg/apis/migration/v1alpha1"
+	gvk "github.com/fusor/mig-controller/pkg/gvkcompare"
+)
+
+func (r ReconcileMigPlan) compareGVK(plan *migapi.MigPlan) error {
+	// No spec chage this time
+	if plan.HasReconciled() {
+		return nil
+	}
+
+	plan.Status.InitUnsupported()
+
+	gvkCompare, err := r.prepareGVKCompare(plan)
+	if err != nil {
+		log.Error(err, "Failed to prepare GVK compare")
+		return err
+	}
+
+	err = gvkCompare.Compare()
+	if err != nil {
+		log.Error(err, "Failed to compare GVRs on both clusters")
+		return err
+	}
+
+	if len(plan.Status.UnsupportedNamespaces) > 0 {
+		plan.Status.SetCondition(migapi.Condition{
+			Type:     NotSupported,
+			Status:   True,
+			Category: Warn,
```

Comment:
Note to anyone else reviewing, this condition is a `Warn` at present. 
cc @jortel